frontend/src/composables/useShootSpec.js

```diff
@@ -119,11 +121,20 @@ export function useShootSpec (shootItem, options = {}) {
   })
 
   const isCustomShootDomain = computed(() => {
-    return some(shootDnsProviders.value, 'primary')
+    return !!shootDnsPrimaryProvider.value
   })
 
-  const shootDnsProviders = computed(() => {
-    return get(shootSpec.value, 'dns.providers')
+  const shootDnsPrimaryProvider = computed(() => {
```

Comment:
We do you delete this computed property and not do something like this:
```js
 const shootDnsProviders = computed(() => {
    return get(shootSpec.value, 'dns.providers')
  })
  
  const shootDnsPrimaryProvider = computed(() => {
    return find(shootDnsProviders.value, 'primary')
  })
  
 const isCustomShootDomain = computed(() => {
    return !!shootDnsPrimaryProvider.value
  })
```

Reply:
we no longer need `shootDnsProviders` as we read additional DNS providers from `spec.extensions`. We only care about the primary provider. For historical reasons this is a list but in the future only one entry (primary provider) will be allowed.